multimodal-search-pdf/flows/query.yml

```diff
@@ -0,0 +1,60 @@
+jtype: Flow
+version: '1'
+with:
+  workspace: $JINA_WORKSPACE
+  read_only: true
```

Comment:
Need to add the port here to alwys use the sam port: 
```port_expose: $JINA_PORT```

Reply:
https://github.com/jina-ai/examples/pull/629/commits/999c9167f385ca39431b7b3fa8d15b1881e79874